### ISSUE TITLE
Incorrect datetime value: '1972-11-09 00:00:00' for function str_to_date

### ISSUE THREAD

**Issue by sepastian** [DIA-NZ/webcurator#113]
Hi, I'm trying to change bandwith settings in a fresh install, and receive the following error.

```
org.springframework.jdbc.UncategorizedSQLException: Hibernate operation: Could not execute JDBC batch update; uncategorized SQLException for SQL [insert into DB_WCT.BANDWIDTH_RESTRICTIONS (BR_BANDWIDTH, BR_DAY, BR_END_TIME, BR_START_TIME, BR_OPTIMIZATION_ALLOWED, BR_OID) values (?, ?, ?, ?, ?, ?)]; SQL state [HY000]; error code [1411]; Incorrect datetime value: '1972-11-09 00:00:00' for function str_to_date; nested exception is java.sql.BatchUpdateException: Incorrect datetime value: '1972-11-09 00:00:00' for function str_to_date
```

**Comment by hannakoppelaar** [comment]
This looks like a problem in the constraint definition. This used to not be a problem, but since MariaDB 10.2.1 these constraints are actually checked. I will try to fix this in one go with #114.

**Comment by hannakoppelaar** [comment]
As a manual workaround for now you could execute these commands in the mysql client:

```
alter table BANDWIDTH_RESTRICTIONS drop constraint CHK_END_TIME;
alter table BANDWIDTH_RESTRICTIONS drop constraint CHK_START_TIME;
alter table DB_WCT.BANDWIDTH_RESTRICTIONS add constraint CHK_END_TIME check (br_end_time >= STR_TO_DATE('1972-11-09 00:00:00', '%Y-%m-%d %H:%i:%s') AND br_end_time <= STR_TO_DATE('1972-11-09 23:59:59', '%Y-%m-%d %H:%i:%s'));
alter table DB_WCT.BANDWIDTH_RESTRICTIONS add constraint CHK_START_TIME check (br_start_time >= STR_TO_DATE('1972-11-09 00:00:00', '%Y-%m-%d %H:%i:%s') AND br_start_time <= STR_TO_DATE('1972-11-09 23:59:59', '%Y-%m-%d %H:%i:%s'));
```
